masterlist.yaml

```
@@ -8453,6 +8453,17 @@ plugins:
     group: *earlyLoadersGroup
     tag: [ Actors.ACBS ]
 
+  - name: 'ProtectUniqueNPCs.esp'
+    url:
+      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/49667/'
+        name: 'Protect Unique NPCs'
+    group: *earlyLoadersGroup
+    inc:
+      - 'NPCs Protected Redux.esp'
+      - 'NPCs Protected and Uncapped.esp'
```

These should be sorted alphabetically:

```suggestion
      - 'NPCs Protected and Uncapped.esp'
      - 'NPCs Protected Redux.esp'
```